service/routes.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"github.com/YuvalSteiy/book_service/bl"
+	"github.com/gin-gonic/gin"
+)
+
+func ConfigRoutes(router *gin.Engine){
+	router.GET("/book/:id", bl.GetBookByID)
+	router.PUT("/book", bl.InsertBook)
+	router.POST("/book/:id", bl.UpdateBook )
+	router.DELETE("/book/:id", bl.DeleteBook )
+	router.GET("/search" , bl.SearchBook )
+	router.GET("/store", bl.GetStoreInfo)
+	router.GET("/activity", bl.GetUserData)
```

Comment:
I'm getting empty results (the server is returning status `200`)
![Screen Shot 2021-06-10 at 16 32 43](https://user-images.githubusercontent.com/25406936/121534003-7ddbb800-ca09-11eb-8770-56b33c84847d.png)
![Screen Shot 2021-06-10 at 16 32 53](https://user-images.githubusercontent.com/25406936/121534039-8502c600-ca09-11eb-9976-a56080be106f.png)

Reply:
it definitely works for me here. and logic seems to make sense...
What do you suggest we do?

Reply:
<img width="1083" alt="Screen Shot 2021-06-13 at 9 50 16" src="https://user-images.githubusercontent.com/83537953/121798055-c45d2c80-cc2c-11eb-8155-fe1b0d22bdd0.png">